restore/harbor/README.md

```diff
@@ -45,6 +45,31 @@ envsubst > tmp-job.yaml < restore/harbor/restore-harbor-job.yaml
 
 ## Restore job Azure
 
+### Rclone move data
+
+If you are restoring Harbor in Azure from a bucket that was previously a rclone destination where the source was not Azure, then the path for the Harbor image data is likely wrong.
+Due to an issue with Harbor it is storing image data with an extra `/` at the start of the path for Azure.
```

Comment:
NIT: Maybe we should add a link to the distributor issue here so it's easy to find

Reply:
Yes, thanks.